src/game/game.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Battleships game</title>
+</head>
+<body>
+    <main class="wrapper">
+        <div class="wrapper__main">
+            <button class="startGameButton">Let's play!</button>
+            <div class="timer">
+                <span class="timer__counter">10:00</span>
+            </div>
+            <div class="board__wrapper">
+                <div class="board__player">
+                    <h1 class="board__player-name player">Player</h1>
```

Comment:
Change tag `<h1>`. Only one max should occur per page and you have already two for player and PC name.